web/app/controllers/dashboard_controller.rb

```diff
@@ -24,6 +31,30 @@ def index
 
   private
 
+  def get_application_name(app_id)
+    req = Soapbox::GetApplicationRequest.new(id: app_id)
+    @app = $api_client.get_application(req)
+    @app.name
+  end
+
+  def get_activities
+    @activities = $api_activity_client
+    .list_activities(Soapbox::Empty.new)
```

Comment:
indent this and the next line to indicate that this is all one logical statement